### PR TITLE
Add call for elections

### PR DESCRIPTION
**Pull request by lpugin**

### CODE REVIEW

File: _posts/2021-11-20-elections.md
Comment (musicEnfanthen) [inline _posts/2021-11-20-elections.md:7]:
```suggestion
On 31 December 2021 the terms of three MEI Board members will come to an end. The entire Board wishes to thank Benjamin W. Bohl, Elsa De Luca, and Ichiro Fujinaga for their service and dedication to the MEI community. Additional gratitude is due Elsa for her leadership as Chair of the Board.
```

**Review by musicEnfanthen — COMMENTED** [review]
Thank you, looks good. See one comment.

**Review by musicEnfanthen — APPROVED** [review]
Thank you!

**Comment by musicEnfanthen** [comment]
@lpugin Shall we wait for another review that you requested? Otherwise, looks good to merge for me.

**Comment by lpugin** [comment]
No need to wait for me. Thanks!

**Pull request merged**
